OSSSpeechKit/Classes/OSSSpeech.swift

```diff
@@ -280,6 +280,21 @@ public class OSSSpeech: NSObject {
         speak()
     }
     
+    /// Stop speaking text
+    public func stopSpeaking() {
+        speechSynthesizer.stopSpeaking(at: .immediate)
+    }
+    
+    /// Pause speaking text
+    public func pauseSpeaking() {
+        speechSynthesizer.pauseSpeaking(at: .immediate)
+    }
+    
+    /// Continue speaking text
+    public func continueSpeaking() {
+        speechSynthesizer.continueSpeaking()
```

Comment:
Like the above, but instead of checking `isSpeaking` you need to check `isPaused` value and return the BOOL value.